packages/fxa-auth-server/lib/senders/emails/partials/verificationReminder/index.ts

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
The base template for both `verificationReminderFirst` and `verificationReminderSecond` emails remains the same so I created a separate partial `verificationReminder` for it and just passing down the templatevalues to it while rendering the emails